### PR TITLE
Update stale bot configuration

### PR DESCRIPTION
**Pull request by Aaron-Junker**
* Never close PRs
* Never close issues with label "bug", "dependency" and "accessibility".
* Also shorten the time until an issue is closed.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -17,20 +17,19 @@ jobs:
...
-          days-before-stale: 14
-          days-before-close: 5
-          exempt-issue-labels: "keep-me"
+            The PR will be closed in 2 days. If this is incorrect,
+            simply comment on the PR, or remove the stale label.
+          days-before-stale: 7
+          days-before-close: 2
+          exempt-issue-labels: "bug,dependency,accessibility"
           exempt-pr-labels: "keep-me"
+          close-issue-reason: "not_planed"
```
Comment (jeffwilcox) [inline .github/workflows/stale.yml:34]: not_planned?

**Review by jeffwilcox — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 1 above)
Comment (Aaron-Junker) [reply .github/workflows/stale.yml:34]: This closes the issues with the status "not planned".

image

Comment (jeffwilcox) [reply .github/workflows/stale.yml:34]: But is it misppelled as "planed" in the value? I'd expect this to be "not_planned" unless there's a typo in the API...

Comment (Aaron-Junker) [reply .github/workflows/stale.yml]: You're right. I fixed it.

**Review by jeffwilcox — APPROVED** [review]

**Pull request merged**